stdlib/public/Darwin/Foundation/String.swift

```diff
@@ -54,10 +73,47 @@ extension String : _ObjectiveCBridgeable {
   public static func _unconditionallyBridgeFromObjectiveC(
     _ source: NSString?
   ) -> String {
+    if let source = source {
+      #if !(arch(i386) || arch(arm))
```

Comment:
Isn't `_bridgeTaggedCocoaString` guarded?

Reply:
It is, but here we definitely don't get to lean on the inliner. Maybe I should just guard the entire definition of _bridgeTaggedCocoaString so it doesn't even exist on 32 bit?